Lombiq.HelpfulLibraries.OrchardCore/ResourceManagement/ResourceFilterMiddleware.cs

```diff
@@ -53,9 +54,18 @@ public async Task InvokeAsync(HttpContext context)
                         ? Task.FromResult(filter.Filter(context))
                         : filter.FilterAsync(context));
 
-            activeFilters.ForEach(
-                filter => filter.Execution(resourceManager),
-                _ => resourceManager = context.RequestServices.GetService<IResourceManager>());
+            foreach (var filter in activeFilters)
+            {
+                resourceManager ??= context.RequestServices.GetService<IResourceManager>();
```

Comment:
The `.GetService()` could in theory return null. Won't the `ExecutionAsync(resourceManager)` fail if this happened?
Might be better to use `.GetRequiredService()`